importer/api/__init__.py

```diff
@@ -144,6 +144,19 @@ def import_course(bundle, repo_id, user_id, static_dir):
     return course
 
 
+def is_leaf_tag(tag):
+    """
+    Should we look for resources within elements with this tag?
+
+    Args:
+        tag (unicode): Element tag
+    Returns:
+        bool: Whether tag is leaf tag
+    """
+    return tag in ('video', 'html', 'problem', 'discussion')
```

Comment:
I'm not sure it's more performant for four elements but it's probably cleaner. By the way this can be formatted this as a set literal instead of using `set()`: `{'video', 'html', 'problem', 'discussion'}`